seed/main/bho.xml

```diff
@@ -20,7 +20,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 	</localeDisplayNames>
 	<characters>
 		<exemplarCharacters draft="contributed">[क ख ग घ डं च छ ज झ ञ ट ठ ड ढ ण ड़ ढ़ त थ द ध न प फ ब भ म य र ल व श ष स ह क्ष त्र ज्ञ श्र ा  ि  ी  ु  ू  ृ  े  ै  ो  ौ  ँ  ं  ः  ॅ  ्]</exemplarCharacters>
-		<exemplarCharacters type="auxiliary">[अ आ इ ई उ ऊ ऋ ए ऐ ओ औ अ:]</exemplarCharacters>
+		<exemplarCharacters type="auxiliary">[अ आ इ ई उ ऊ ऋ ए ऐ ओ औ]</exemplarCharacters>
```

Comment:
I wonder whether that character with colon in aux was actually supposed to have \u0903 DEVANAGARI VISARGA instead, which is a combining char that looks like colon

Reply:
(But \u0903 is already in the main exemplars so no need to have the combination in aux)

Reply:
Makes sense!